package.json

```diff
@@ -12,8 +12,7 @@
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
-    "prepublishOnly": "rm -rf ./dist && babel ./src --out-dir ./dist -s inline",
-    "postpublish": "git tag v$npm_package_version && git push origin v$npm_package_version"
```

Comment:
I often remove this while testing, but do we want to remove this permanently?

Reply:
Yes, if we use yarn this is useless because `yarn publish` automatically  creates a tag